src/main/java/com/conveyal/datatools/manager/models/Model.java

```diff
@@ -39,6 +39,8 @@ public Model () {
 
     public String id;
 
+    public String retrieveId() { return this.id; }
```

Comment:
This should be named `getId()`.

Reply:
(and `retrieveNotes()` should be renamed `getNotes()`)

Reply:
Getters unfortunately cannot be named `getId` due to issues with the Mongo codecs (same with setters). Perhaps there should be comments on these `retrieve*` methods instead.

Reply:
I could just remove the getter and change the lambda syntax.